client/components/map.jsx

```diff
@@ -71,82 +56,75 @@ export default function Map(props) {
   const onMapLoad = React.useCallback(map => {
     mapRef.current = map;
   }, []);
-
   if (loadError) return 'Error loading maps';
   if (!isLoaded) return 'Loading Maps';
 
   const handleAddSoundscapeClick = () => {
     setModal(true);
   };
 
-  return <div>
-    <GoogleMap
-    mapContainerStyle={mapContainerStyle}
-    zoom={7}
-    center={center}
-    onClick={onMapClick}
-    onLoad={onMapLoad}
-    >
-
-      {markers.map(markers => (
-        <Marker
-          key={markers.soundscapeId}
-          position={{ lat: markers.lat, lng: markers.lng }}
-        onClick={() => {
-          setSelectedMarker(markers);
-        }}
-        />
-      ))}
-
-      {selectedMarker &&
-          <InfoWindow
-          position={{ lat: selectedMarker.lat, lng: selectedMarker.lng }}
-            onCloseClick={() => { setSelectedMarker(null); }}>
-            <div>
-              <h5>Title: {selectedMarker.title}</h5>
-              <p>Description: {selectedMarker.description}</p>
-              <div className={styles.player}>
-                <audio ref={audioPlayer} src={selectedMarker.fileUrl}></audio>
-                <button size='sm' className={styles.circle} onClick={togglePlayPause}>
-                  {isPlaying
-                    ? <PauseBtnFill className={styles.play}/>
-                    : <PlayBtnFill className={styles.pause} />
-                  }
-                </button>
+  return (
+    <div>
+      <GoogleMap
+      mapContainerStyle={mapContainerStyle}
+      zoom={7}
+      center={center}
+      onClick={onMapClick}
+      onLoad={onMapLoad}
+      >
+
+        {markers.map(markers => (
+          <Marker
+            key={markers.soundscapeId}
+            position={{ lat: markers.lat, lng: markers.lng }}
+            onClick={() => {
+              setSelectedMarker(markers);
+            }}
+          />
+        ))}
```

Comment:
```js
{markers.map(markers => (
```

This implies that each element of the `markers` array is a "markers", which is confusing. Try renaming this parameter to just `marker`.